python/plugins/processing/algs/qgis/QGISAlgorithmProvider.py

```diff
@@ -78,6 +78,7 @@
 from .VectorSplit import VectorSplit
 from .VoronoiPolygons import VoronoiPolygons
 from .ZonalStatistics import ZonalStatistics
+from .Heatmap import Heatmap
```

Comment:
Mind keeping the alphabetical sorting that's been started here? (and below)

(sorry, pedantic I know!)